scripts/screeps-runtime-monitor.py

```
@@ -779,6 +809,16 @@ def env_default(name: str, default: str) -> str:
     return os.environ[name] if name in os.environ else default
 
 
+def positive_float_arg(value: str) -> float:
+    try:
+        parsed = float(value)
+    except ValueError as exc:
+        raise argparse.ArgumentTypeError(f"expected a positive number, got {value!r}") from exc
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)